specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```diff
@@ -862,6 +862,9 @@
           "description": "Version",
           "type": "string",
           "example": "2.0"
+        },
+        "storageSource": {
+          "$ref": "#/definitions/StorageSource"
```

Comment:
Is this addition of new property is intentional? I see the description says PR is about doc update, so just want to make sure addition of this property is not by error.

Reply:
or this property is to enable the file formats?

Reply:
yes this is an intentional property, but it is completely backward compatible (this property has a default value keeping the same behavior)

Reply:
updated the PR title